CHANGELOG.unreleased.md

```diff
@@ -17,7 +17,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
-[XO6/New VM] Display only ISO VDIs for the ISO input (PR [#8922](https://github.com/vatesfr/pull/8922))
+- [XO6/New VM] Display only ISO VDIs for the ISO input (PR [#8922](https://github.com/vatesfr/pull/8922))
+- [V2V] fix resuming importing retransfering the full VM (PR [#](https://github.com/vatesfr/pull/))
```

Comment:
I don't think this changelog should be there
```suggestion
```

Reply:
https://github.com/vatesfr/xen-orchestra/pull/8933